client/index.jsx

```diff
@@ -3,6 +3,29 @@ import ReactDOM from 'react-dom';
 import App from './app';
 
 ReactDOM.render(
-  <App />,
+  <>
+    <nav>
+      <div className="nav-wrapper bg-columbia-blue">
+        <a href="#" className="brand-logo center bright-gray">Rooms</a>
+      </div>
+    </nav>
+  </>,
+  document.querySelector('header')
+);
+
+ReactDOM.render(
+  <>
+    <div className="container">
+      <div className="row">
+      </div>
+    </div>
+  </>,
+  document.querySelector('footer')
+);
```

Comment:
We don't usually do multiple roots if we can avoid them. Instead App should render a "shell" around the page content like in `react-hash-routing` or `react-context`